src/main/java/core/basesyntax/model/ma/Person.java

```diff
@@ -1,9 +1,26 @@
 package core.basesyntax.model.ma;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
```

Comment:
https://github.com/mate-academy/hibernate-inheritance-example
Take a look what specific inheritance type we should use in every of four specific cases